web/src/containers/activity/NonPersonnelCost/NonPersonnelCostForm.js

```diff
@@ -0,0 +1,95 @@
+import {
+  Button,
+  FormLabel,
+  Select,
+  TextField
+} from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React, { useCallback } from 'react';
+
+import DollarField from '../../../components/DollarField';
+
+const NonPersonnelCostForm = ({
+  category,
+  collapse,
+  desc,
+  handleEditCost,
+  handleEditDesc,
+  handleEditCategory,
+  idx,
+  years
+}) => {
+  const editCategory = useCallback(
+    ({ target: { value } }) => handleEditCategory(idx, value),
+    [idx]
+  );
+
+  const editDesc = useCallback(
+    ({ target: { value } }) => handleEditDesc(idx, value),
+    [idx]
+  );
+
+  const getEditCostForYear = useCallback(
+    year => ({ target: { value } }) => handleEditCost(idx, year, value),
+    [idx]
+  );
+
+  return (
+    <div className="ds-u-border-bottom--2 ds-u-padding-y--2">
```

Comment:
It occurred to me as I was wrapping this one up that we can have another HUC to further wrap up this pattern because we have it a lot.  It's easier to make _one_ thing consistent than to make a dozen things consistent.  😬

My thinking is something like:

```js
<ListOfThingsthings
    things={array}
    collapsed={ThingReview}
    expanded={ThingForm}
/>
```

And let that `ListOfThings` container handle the lines and everything and then it's the same everywhere.  Right now some of the review containers have lines, some of the forms have lines, and some of the outer things have lines, and it's just kind of a mess.  This would also mean we'd just have one of those collapse manager HUCs instead of one for each kind of thing.  I'm leaning towards this being a ***VERY*** worthwhile short-term refactor because otherwise we may never get a handle on this stuff.